config/crd/bases/atlas.mongodb.com_atlasclusters.yaml

```diff
@@ -166,7 +166,6 @@ spec:
                     type: string
                 required:
                 - name
-                - namespace
```

Comment:
have we changed something that's inadvertently affected these Atlas models (this and atlasdatabaseusers) this way?

Reply:
Well, this is the result of code generation, since namespace is set here in the type definition:
https://github.com/mongodb/mongodb-atlas-kubernetes/blob/main/pkg/api/v1/common.go#L23

What do you think we should do?

Reply:
As discussed we will keep it as it is.